client/layout/utils.ts

```diff
@@ -1,20 +1,3 @@
-const HOUR_IN_MS = 60 * 60 * 1000;
-
-/**
- * Returns false if the site is unlaunched or is younger than 1 hour
- *
- * @param site the site object
- */
-export function getShouldShowAppBanner( site: any ): boolean {
-	if ( site && site.options ) {
-		const olderThanAnHour = Date.now() - new Date( site.options.created_at ).getTime() > HOUR_IN_MS;
```

Comment:
This PR also removes the condition that the site must be at least one hour old. Reading the #54948 issue, it seems to me that condition should stay.

The purpose is to not distract the new user from working on their new site by offering the mobile app. That can be done later, when the banner doesn't interrupt the current flow.